### PR TITLE
bugfix: update mailing list to inlong.apache.org and issue url

### PR DESCRIPTION
**Pull request by leven-space**

### CODE REVIEW

File: docs/en-us/contact.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ Contact us
...
     |:------------------------------------------------------------------------------|:--------------------------------|:----------------------------------------------------------------|:--------------------------------------------------------------------|:-----------------------------------------------------------------------------|
-    | [dev@tubemq.apache.org](mailto:dev@tubemq.apache.org)     | Development-related discussions | [Subscribe](mailto:dev-subscribe@tubemq.apache.org)   | [Unsubscribe](mailto:dev-unsubscribe@tubemq.apache.org)   | [Archives](http://mail-archives.apache.org/mod_mbox/tubemq-dev/)   |
+    | [dev@tubemq.apache.org](mailto:dev@inlong.apache.org)     | Development-related discussions | [Subscribe](mailto:dev-subscribe@inlong.apache.org)   | [Unsubscribe](mailto:dev-unsubscribe@inlong.apache.org)   | [Archives](http://mail-archives.apache.org/mod_mbox/tubemq-dev/)   |
```
Comment (RocMarshal) [inline docs/en-us/contact.md:12]: Thank you for your efforts. Would you please check the address of Archives in this issue? Same for the following

Comment (leven-space) [reply docs/en-us/contact.md:12]: Hi，I noticed that tubemq-dev archives's address still working. The original mail was not migrated~

Comment (RocMarshal) [reply docs/en-us/contact.md:12]: LGTM. Now ping @gosonzhang .

Comment (gosonzhang) [reply docs/en-us/contact.md:12]: LGTM

**Review by gosonzhang — APPROVED** [review]
LGTM

**Pull request merged**